src/app/learn/get-started/advanced/localization/index.html

```diff
@@ -4,18 +4,154 @@
 
   <stache-page-summary>
     <p>
-      SKY UX Builder provides the capability to declare localization strings <a routerLink="/learn/reference/naming-and-placement">in a JSON file </a> and then reference those strings throughout your SPA using the <stache-code>skyAppResources</stache-code> pipe or the <stache-code>SkyAppResourcesService</stache-code> service in <stache-code>@skyux/i18n</stache-code>.
+      SKY UX Builder provides two methods for localization, <stache-code>skyLibResources</stache-code> for libraries and <stache-code>skyAppResources</stache-code> for SPAs. 
     </p>
   </stache-page-summary>
 
   <sky-alert alertType="info">
     SKY UX Builder version <stache-code>1.14</stache-code> or later is required for a library to provide its own resource files. The consuming SPA must also be on version <stache-code>1.14</stache-code> or later.
   </sky-alert>
 
+<stache-page-anchor>
+  Localization for libraries
+</stache-page-anchor>
+
+<p>
+  Localization strings for libraries must be set when publishing to NPM because they are not fetched at runtime as they are for SPAs. 
+</p>
+
+<h3>
+  Install the plugin
+</h3>
+
+<ol>
+  <li>
+    <p>
+      Install the SKY UX builder plugin.
+    </p>
+      <sky-code-block>
+        npm install --save-dev --save-exact @skyux-sdk/builder-plugin-skyux
+      </sky-code-block>
+    </li>
+  <li>
+    <p>
+      Add the plugin to your <stache-code>skyuxconfig.json file</stache-code>.
+    </p>
+      <sky-code-block>
+        {
+          "plugins": [
+            "@skyux-sdk/builder-plugin-skyux"
+          ]
+        }  
+      </sky-code-block>
+  </li>
+</ol>
+
+<h3>
+  Create a resources provider
+</h3>
+
+  <ol>
+    <li>
+      <p>
+        Create a new directory named <stache-code>plugin-resources</stache-code> in <stache-code>./src/app/public/plugin-resources</stache-code>. The name and location of the directory are important.
```

Comment:
I'd delete "new" because it's redundant. I'd also change the start to "Create a `plugin-resources` directory in..."

Also, I'd delete the period before `/src` in the folder path. We don't usually have a period in front of `/src` in our paths.